luigi/scheduler.py

```diff
@@ -556,6 +556,12 @@ def get_worker_ids(self):
     def get_worker(self, worker_id):
         return self._active_workers.setdefault(worker_id, Worker(worker_id))
 
+    def get_worker_starts(self):
```

Comment:
what is this??? Does it mean `get_started_workers`? If you can't rename it to something self-describing. Can you add a comment instead?